src/utils/RouteConfig.js

```diff
@@ -69,11 +70,11 @@ export const routes = [
   { name: translate('topology'), slug: '/topology',
     items: [
       { name: translate('control_planes'), slug: '/topology/control_planes', component: ControlPlanes },
-      { name: translate('regions'), slug: '/topology/regions', component: Regions },
-      { name: translate('services'), slug: '/topology/services', component: Example },
-      { name: translate('networks'), slug: '/topology/networks', component: Example },
-      { name: translate('servers'), slug: '/topology/servers', component: Example },
-      { name: translate('server_groups'), slug: '/topology/server-groups', component: Example },
+      { name: translate('regions'), slug: '/topology/regions', component: Example },
```

Comment:
git merged this the wrong way for what you want, that explains why the example content is showing up instead of the region topology